packages/core/src/types.ts

```diff
@@ -82,3 +82,6 @@ export type RequireOnlyOne<T, Keys extends keyof T = keyof T> = Pick<T, Exclude<
   {
     [K in Keys]-?: Required<Pick<T, K>> & Partial<Record<Exclude<Keys, K>, undefined>>;
   }[Keys];
+
+export type AudioSource = Track.Source.Microphone | Track.Source.ScreenShareAudio;
+export type VideoSource = Track.Source.Camera | Track.Source.ScreenShare;
```

Comment:
I had included it at first, but removed, because I thought it might be better to make it explicit for those types. For `Unknown` we do not know by definition whether it is an AudioSource or a VideoSource.
The advantage of leaving it out, is that we avoid overlap